prowler/providers/aws/services/organizations/organizations_service.py

```diff
@@ -0,0 +1,233 @@
+
+
+################## Organizations
+class Organizations:
+    def __init__(self, audit_info):
+        self.service = "organizations"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audit_resources = audit_info.audit_resources
+        global_client = generate_regional_clients(
+            self.service, audit_info, global_service=True
+        )
+        self.client = list(global_client.values())[0]
+        self.region = self.client.region
+        self.organizations = []
+        self.policies = []
+        self.delegated_administrators = []
+        self.__describe_organization__()
+
+    def __describe_organization__(self):
+        logger.info("Organizations - Describe Organization...")
+
+        try:
+            # Check if Organizations is in-use
+            try:
+                organization_desc = self.client.describe_organization()["Organization"]
+                organization_arn = organization_desc.get("Arn")
+                organization_id = organization_desc.get("Id")
+                organization_master_id = organization_desc.get("MasterAccountId")
+                organization_available_policy_types = organization_desc[
+                    "AvailablePolicyTypes"
+                ]
+                # Fetch policies for organization:
+                organization_policies = self.__list_policies__(
+                    organization_available_policy_types
+                )
+                # Fetch delegated administrators for organization:
+                organization_delegated_administrator = (
+                    self.__list_delegated_administrators__()
+                )
+            except ClientError as e:
+                if e.response["Error"]["Code"] == "AWSOrganizationsNotInUseException":
+                    self.organizations.append(
+                        Organization(
+                            arn="", id="", status="NOT_AVAILABLE", master_id=""
+                        )
+                    )
+            else:
+                if not self.audit_resources or (
+                    is_resource_filtered(organization_arn, self.audit_resources)
+                ):
+                    self.organizations.append(
+                        Organization(
+                            arn=organization_arn,
+                            id=organization_id,
+                            status="ACTIVE",
+                            master_id=organization_master_id,
+                            policies=organization_policies,
+                            delegated_administrators=organization_delegated_administrator,
+                        )
+                    )
+                else:
+                    # is filtered
+                    self.organizations.append(
+                        Organization(
+                            arn="", id="", status="NOT_AVAILABLE", master_id=""
```

Comment:
If we are creating this dummy object here we cannot leave these fields as empty strings since all check's findings should have at least the ID or the ARN to be filled.

Check the dummy SecurityHub Hub:
```python
SecurityHubHub(
                            arn="",
                            id="Security Hub",
                            status="NOT_AVAILABLE",
                            standards="",
                            integrations="",
                            region=regional_client.region,
                        )
```

Reply:
i fixed this.